storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLiteTest.java

```diff
@@ -242,6 +242,39 @@ public void typeMappingShouldFindIndirectTypeMappingInCaseOfComplexInheritance()
         assertThat(storIOSQLite.internal().typeMapping(AutoValue_ConcreteEntity.class)).isSameAs(concreteEntitySQLiteTypeMapping);
     }
 
+    interface InterfaceEntity {
+    }
+
+    @Test
+    public void typeMappingShouldFindIndirectTypeMappingForInterfaces() {
+        class ConcreteEntity implements InterfaceEntity {
+        }
+
+        class Parent_ConcreteEntity extends ConcreteEntity {
+        }
+
+        //noinspection unchecked
+        final SQLiteTypeMapping<InterfaceEntity> interfaceSQLiteTypeMapping = SQLiteTypeMapping.<InterfaceEntity>builder()
+                .putResolver(mock(PutResolver.class))
+                .getResolver(mock(GetResolver.class))
+                .deleteResolver(mock(DeleteResolver.class))
+                .build();
+
+        final StorIOSQLite storIOSQLite = DefaultStorIOSQLite.builder()
+                .sqliteOpenHelper(mock(SQLiteOpenHelper.class))
+                .addTypeMapping(InterfaceEntity.class, interfaceSQLiteTypeMapping)
+                .build();
+
+        // Direct type mapping for InterfaceEntity should work
+        assertThat(storIOSQLite.internal().typeMapping(InterfaceEntity.class)).isSameAs(interfaceSQLiteTypeMapping);
+
+        // Indirect type mapping for InterfaceEntity should work
+        assertThat(storIOSQLite.internal().typeMapping(ConcreteEntity.class)).isSameAs(interfaceSQLiteTypeMapping);
```

Comment:
Can you please extract this into separate test so it would be easier for us to support this in future?